src/app/fundraisings/[fundraisingId]/jars/JarsList/AddJarDialog/AddJarDialog.tsx

```diff
@@ -57,7 +60,8 @@ export const AddJarDialog = ({
     const createJarPayload: CreateJarPayload = {
       url,
       ownerName: owner,
-      parentJarId: Number(formData.get('parentJarId')),
+      fundraisingCampaignId: +fundraisingId,
```

Comment:
While `+stringVariable` is good, I prefer `Number` because it's more verbose. So let's please stick to convention